packages/design-tokens/src/tokens/functional/components/timeline/base.json

```
@@ -2,7 +2,7 @@
   "brand": {
     "Timeline": {
       "lineHeight": {
-        "value": "var(--brand-text-lineHeight-300)"
+        "value": "var(--base-size-16)"
```

```suggestion
        "value": "var(--brand-text-lineHeight-200)"
```
Could we keep this one in sync with the font-size [set in the component](https://github.com/primer/brand/blob/dced12b62701bad2dbb7780bb690b15f16dd9069/packages/react/src/Timeline/Timeline.tsx#L45)? This way we wouldn't have to manually adjust the position of the bullet as [introduced here](https://github.com/primer/brand/blame/0e131f26b3437f698f0e0c4118cc26e3a4b09f2c/packages/react/src/Timeline/Timeline.module.css#L2).